ui/v2.5/src/components/Scenes/SceneDetails/RatingSystem.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useRef } from "react";
+import * as GQL from "src/core/generated-graphql";
+import { ConfigurationContext } from "src/hooks/Config";
+import Box from "@mui/material/Box";
+import StarIcon from "@mui/icons-material/Star";
+import StarBorderIcon from "@mui/icons-material/StarOutline";
+
+export interface IRatingSystemProps {
+  value?: number;
+  onSetRating?: (value?: number) => void;
+  disabled?: boolean;
+}
+
+export interface IRatingStarsProps {
+  value?: number;
```

Comment:
There's technically now a bug here where if you change the rating in the edit menu and switch to the details menu, the details menu will show the old value unless the page is refreshed. This doesn't show up for the number rating but it does for the star rating which is strange. Not currently sure what could cause that.